backend-k8s/steward-system/101-customResourceDefinition_PipelineRun.yaml

```diff
@@ -64,25 +64,21 @@ spec:
                   pattern: '^[^\s]{1,}.*$'
                 repoAuthSecret: ###
                   type: string
-                  pattern: '^[^\s]{1,}.*$'
```

Comment:
I think we should not allow empty names for secrets.

Reply:
Same argument would apply for all other parameters where I had to remove the regex. JSON "", null and parameter not set at all, ends up in "" in the go struct.

Reply:
For `spec.jenkinsFile.repoAuthSecret` the empty string would be ok. For `spec.secrets` and `spec.imagePullSecrets` we don't want empty or null list entries.

Reply:
Agree